podman/domain/volumes.py

```diff
@@ -34,6 +34,16 @@ def remove(self, force: Optional[bool] = None) -> None:
             APIError: when service reports an error
         """
         self.manager.remove(self.name, force=force)
+    
+    def inspect(self) -> dict:
+        """Inspect this volume
+        
+        Raises:
+            APIError: when service reports an error
+        """
```

Comment:
Please, could you something like podman/domain/images_manager.py:L365?
```python
params = {
    "tlsVerify": kwargs.get("tlsVerify"),
}
```

let's try to honor the option in the [docs](https://docs.podman.io/en/latest/_static/api.html#tag/manifests/operation/ManifestInspectLibpod)

Reply:
Good point. Added